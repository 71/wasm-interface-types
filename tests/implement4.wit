;; parse-fail

(module
  (@interface implement (func $foo) (func 1))
  (@interface func $foo)
)

(; CHECK-ALL:
  failed to find func named `$foo`
       --> tests/implement4.wit:4:31
        |
      4 |   (@interface implement (func $foo) (func 1))
        |                               ^

  Backtrace disabled; run with RUST_LIB_BACKTRACE=1 environment variable to display a backtrace
;)
