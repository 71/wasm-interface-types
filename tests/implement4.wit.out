failed to find func named `$foo`
     --> ../../tests/implement4.wit:4:31
      |
    4 |   (@interface implement (func $foo) (func 1))
      |                               ^
